src/main/java/Deadlines.java

```diff
@@ -0,0 +1,22 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task {
+    private LocalDate deadlineTime;
+
+    public Deadlines(String description, LocalDate deadlineTime) {
+        super(description);
+        this.deadlineTime = deadlineTime;
+    }
+
+    @Override
+    public String getStatus() {
+        return "[D]" + super.getStatus() + "(by: " + this.deadlineTime.format(DateTimeFormatter.ofPattern("MMM d yyyy"))
+                + ")";
+    }
+
+    @Override
```

Comment:
Should there be some documentation for this method? I noticed there aren't any javadocs for other methods in the code as well